src/Layouts/Partials/AlignItemsPanel.vala

```diff
@@ -0,0 +1,184 @@
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA
+ *
+ * Authored by: Giacomo "giacomoalbe" Alberini <giacomoalbe@gmail.com>
+ */
+public class Akira.Layouts.Partials.AlignItemsPanel : Gtk.Box {
+  public weak Akira.Window window { get; construct; }
+
+  private Gtk.Box alignmentBox;
+  private Gtk.Box outerBox;
+
+
+  public AlignItemsPanel (Akira.Window main_window) {
+    Object(
+      window: main_window,
+      orientation: Gtk.Orientation.VERTICAL
+    );
+  }
+
+  construct {
+    vexpand = false;
+    hexpand = true;
+
+    outerBox = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
+
+    outerBox.hexpand = true;
+    outerBox.get_style_context ().add_class("alignment-box");
+
+    alignmentBox = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
+    alignmentBox.halign = Gtk.Align.CENTER;
+
+    AlignBoxItem[] alignBoxItems = {
+      AlignBoxItem("btn", "even-h", "distribute-horizontal-center", "Distribute centers evenly horizontally"),
+      AlignBoxItem("btn", "even-v", "distribute-vertical-center", "Distribute center evenly vertically"),
+      AlignBoxItem("sep"),
+      AlignBoxItem("btn", "alig-h-l", "align-horizontal-left", "Align left sides"),
+      AlignBoxItem("btn", "alig-h-c", "align-horizontal-center", "Center on horizontal axis"),
+      AlignBoxItem("btn", "alig-h-r", "align-horizontal-right", "Align right sides"),
+      AlignBoxItem("sep"),
+      AlignBoxItem("btn", "alig-v-t", "align-vertical-top", "Align top sides"),
+      AlignBoxItem("btn", "alig-v-c", "align-vertical-center", "Center on vertical axis"),
+      AlignBoxItem("btn", "alig-v-b", "align-vertical-bottom", "Align bottom sides")
+    };
+
+    foreach (var item in alignBoxItems) {
+      switch (item.type) {
+        case "sep":
+          alignmentBox.pack_start ( new Gtk.Separator (Gtk.Orientation.VERTICAL), false, false, 0);
+          break;
+
+        case "btn":
+          var tmpAlignBoxButton = new Akira.Partials.AlignBoxButton (item.action, item.icon_name, item.title);
+
+          tmpAlignBoxButton.triggered.connect (on_button_event);
+
+          alignmentBox.pack_start (tmpAlignBoxButton, false, false, 0);
+          break;
+      }
+    }
+
+    outerBox.pack_start (alignmentBox);
+
+    pack_start (outerBox);
+
+    connect_signals();
+  }
+
+  private void update_icons_style(string icon_style) {
+    alignmentBox.foreach((child) => {
+      if (child is Akira.Partials.AlignBoxButton) {
+        var button = (Akira.Partials.AlignBoxButton) child;
+
+        button.change_icon_style();
+      }
+    });
+  }
+
+  private void connect_signals() {
+    event_bus.update_icons_style.connect(() => {
+      update_icons_style(settings.icon_style);
+    });
+  }
+
+  private void on_button_event(Akira.Partials.AlignBoxButton button) {
+    print ("Action Triggered: %s\n", button.action);
+  }
+}
+
+class Akira.Partials.AlignBoxButton : Gtk.Grid {
```

Comment:
Move this class into `src/Partials/` on a dedicate file.
We want to avoid having multiple classes in one file and make it easier to maintain it by keeping files small when we can.